internal/k8shandler/defaults.go

```diff
@@ -19,4 +19,7 @@ var (
 
 	defaultFluentdMemory     resource.Quantity = resource.MustParse("736Mi")
 	defaultFluentdCpuRequest resource.Quantity = resource.MustParse("100m")
+
+	defaultVectorMemory     resource.Quantity = resource.MustParse("500Mi")
```

Comment:
Do you think we need even this much memory?  Seems really high given the underlying change. I wonder if we can default to not requiring any resources